runtime/help/tutorial.md

```diff
@@ -1,33 +1,66 @@
 # Tutorial
 
-This is a brief intro to micro's configuration system that will give some
-simple examples showing how to configure settings, rebind keys, and use
-`init.lua` to configure micro to your liking.
+This is a brief intro to `micro` workflow with simple examples how to configure
+settings, keys, and use `init.lua`.
 
 Hopefully you'll find this useful.
 
-See `> help defaultkeys` for a list an explanation of the default keybindings.
+## Command mode
 
-### Settings
+Press `Ctrl-e` to open micro's command prompt. Typing `help tutorial` will open
+this documentation.
+
+For the rest of the docs `> help tutoral` indicates pressing `Ctrl-e`.
+
+## Default keyboard shortcuts
+
+Enter `> help defaultkeys` for a list of the default keybindings.
+
+## Simple workflow: Move content from one file to another
+
+For this example, we edit micro's own codebase, and move contents between files.
+
+Press `Ctrl-o` and enter filename to open the file in the current view
+(use `Tab` key to help with autocompletion)
+```
+> open internal/config/plugin_manager.go
+```
+Then from the command prompt (`Ctrl-e`) open a second file in a vertical split
+```
+> vsplit internal/config/plugin.go
+```
+Use `Ctrl-w` ("jump to next split" shortcut) to switch to the first file and
+cut the `PluginInfo` structure into clipboard using `Ctrl-x`.
+
+Press `Ctrl-w` again to switch back to second file and paste the clipboard
+content using `Ctrl-v`.
+
+Now press `F2` to save current file and `Ctrl-q` to close it.
+
+To preview the changes, run `git diff` by pressing `Ctrl-b` ("shell mode")
+and entering the command. You will see changes only to the second file
+`plugin.go`, because the first file is not saved yet.
```

Comment:
`git` is rarely installed by default so it might not be the best example to use for the command mode.

Reply:
I can rename the tutorial "Copying `micro` code with `micro`", to make it clear that it is for developers. Trying to squeeze everything in one tutorial is impossible anyways, and I don't know the target auditory that works with `micro` without `git`.

If there is another way to see changes since the file was opened, let's list them here.